core/src/test/java/org/hibernate/ogm/backendtck/associations/onetoone/OneToOneTest.java

```diff
@@ -163,7 +167,18 @@ public void testBidirectionalOneToOneCompositeId() throws Exception {
 
 	@Override
 	protected Class<?>[] getAnnotatedClasses() {
-		return new Class<?>[] {
+
+		if ( SkipByHelper.skipForGridDialect( GridDialectType.ORIENTDB ) ) {
```

Comment:
because OrientDB not supports composite keys. And I off entities which uses composite keys